db/migrate/20240514104324_validate_check_constraint_and_make_cohort_id_not_null_in_participant_declarations.rb

```diff
@@ -0,0 +1,12 @@
+class ValidateCheckConstraintAndMakeCohortIdNotNullInParticipantDeclarations < ActiveRecord::Migration[7.1]
+  def up
+    validate_check_constraint :participant_declarations, name: "participant_declarations_cohort_id_null"
```

Comment:
just to confirm, have we done the backfill on all envs or just prod?

Reply:
Just production so far; I'll do the rest now